src/PageContent/RequestPickup/RequestPickupWrapper.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import RecurringPickupRequest from './RecurringPickupRequest';
+import RequestTypeToggle from './RequestTypeToggle';
+import { DeliveryType } from '../../Enums';
+
+class RequestPickupWrapper extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            type: DeliveryType.EMERGENCY
```

Comment:
I think this `type` should represent what the user is requesting rather than what the button is saying.